src/services/EcoBridge/Arbitrum/ArbitrumBridge.reducer.ts

```diff
@@ -100,9 +100,9 @@ export const createArbitrumSlice = (bridgeId: ArbitrumList) =>
     },
   })
 
-const arbitrumSlices: { [k in ArbitrumList]: ReturnType<typeof createArbitrumSlice> } = {
-  'arbitrum:mainnet': createArbitrumSlice('arbitrum:mainnet'),
-  'arbitrum:testnet': createArbitrumSlice('arbitrum:testnet'),
+const arbitrumSlices: { [k in ArbitrumIdList]: ReturnType<typeof createArbitrumSlice> } = {
+  [BridgeIds.ARBITRUM_MAINNET]: createArbitrumSlice(BridgeIds.ARBITRUM_MAINNET),
+  [BridgeIds.ARBITRUM_TESTNET]: createArbitrumSlice(BridgeIds.ARBITRUM_TESTNET),
```

Comment:
Maybe we should use `BRIDGES. [BRIDGE].id` here. What do you think?

Reply:
This applies to other places we are using `BridgeIds`

Reply:
It's tricky due to `BridgeIdList` being a broader type than, for example, `ArbitrumIdList`:

```ts
export interface EcoBridgeConfig {
  id: BridgeIdList // --> here lies the issue
  name: string
  url: string
}

/* ... */

export const BRIDGES: Record<string, EcoBridgeConfig> = { /* ... */ }
```